lib/plugins/aws/provider.js

```diff
@@ -937,7 +938,7 @@ class AwsProvider {
                 },
               },
               required: ['images'],
-              additionalProperties: false,
+              additionalProperties: true,
```

Comment:
This should still be `false` - it signifies if we accept any additional properties in addition to the ones defined explicitly in schema and we don't want to do that :+1:

Reply:
understood - thx!